implementations/dotnet-json-everything/Program.cs

```diff
@@ -58,7 +58,7 @@
 
 			var runResult = new JsonObject
 			{
-				["seq"] = root["seq"],
+				["seq"] = root["seq"].GetValue<int>(),
```

Comment:
This just gets cast back to a `JsonNode` anyway.  There's no benefit to extracting the `int` except for the side effect of checking that it is one.

Reply:
Without it I got another error. One that said:

```
Unhandled exception. System.InvalidOperationException: The node already has a parent.
   at System.Text.Json.ThrowHelper.ThrowInvalidOperationException_NodeAlreadyHasParent()
   at System.Text.Json.Nodes.JsonNode.AssignParent(JsonNode parent)
   at System.Text.Json.Nodes.JsonObject.<>c__DisplayClass9_0.<SetItem>b__0()
   at System.Text.Json.JsonPropertyDictionary`1.SetValue(String propertyName, T value, Action assignParent)
   at System.Text.Json.Nodes.JsonObject.SetItem(String propertyName, JsonNode value)
   at Program.<Main>$(String[] args) in /source/Program.cs:line 59
```

Is there another way that should have been handled? (Google suggested a yet sillier one to serialize back to a string and deserialize again)

Reply:
Hmm yeah, I understand that, but I wouldn't have expected it.

Nodes hold a reference to the parent, which means they can only exist in a single tree.  Not sure how else to handle that.